specs/AppLifecycle/Activation/AppLifecycle Activation.md

```diff
@@ -344,23 +352,35 @@ Similarly, if the app registers multiple times for startup, this is also
+| RegisterForStartupActivation(<br/> string taskId,<br/> bool isEnabled,<br/> string displayName<br/> ); |                                                              |
+
+
 
 ### Activation unregistration
 
 The new API will include UnregisterForXXXActivation methods for each of the
 supported activation kinds.
 
-| API                                                                        | Description                                                                   |
-| -------------------------------------------------------------------------- | ----------------------------------------------------------------------------- |
-| static void UnregisterForFileTypeActivation(<br> string fileTypes[]<br> ); | **fileTypes**: one or more filetypes that the caller wants to unregister.     |
-| static void UnregisterForProtocolActivation(<br> string name<br> );        | **name**: the previously-registered protocol identifier, eg "foo" or "https". |
-| static void UnregisterForStartupActivation();                              |                                                                               |
+| API                                                          | Description                                                  |
+| ------------------------------------------------------------ | ------------------------------------------------------------ |
+| static void UnregisterForFileTypeActivation(<br> string fileTypes[], <br> string exePath<br> ); | **fileTypes**: one or more filetypes that the caller wants to unregister. <br>**exePath**: path to the executable that was registered for activation. |
+| static void UnregisterForFileTypeActivation(<br> string fileTypes[]<br> ); |                                                              |
+| static void UnregisterForProtocolActivation(<br> string name,<br> string exePath<br> ); | **name**: the previously-registered protocol identifier, eg "foo" or "https". <br>**exePath**: path to the executable that was registered for activation. |
+| static void UnregisterForProtocolActivation(<br> string name<br> ); |                                                              |
+| static void UnregisterForStartupActivation(<br> string exePath<br>); | **exePath**: path to the executable that was registered for activation. |
```

Comment:
I don't think this one makes sense.  Unregister (in all of these cases) I *think* should be based on the type and not the EXE.  So fileTypes[], protocol Scheme 'name', and Startup's taskId.

Reply:
We talked offline, and Unregister still needs the exePath for File and Protocol but not Startup.  Instead it needs taskId.

Reply:
Fixed.